### DISCUSSION TITLE
DOI/admin rights

### DISCUSSION THREAD

**Discussion by mjt320** [OSIPI/DCE-DSC-MRI_CodeCollection#128] [Q&A]
Hi there,

Before submitting the manuscript we were planning to create a release + DOI for the repository using Zenodo.
I'm happy to do this but need to be an admin. @LucyKershaw @stadmill are you able to give me that status temporarily?

Thanks,
Michael

**Comment by LucyKershaw** [comment]
I'd be delighted to do this, if only I knew how. What is zenodo?

**Comment by plaresmedima** [comment]
Hi Michael - is there an OSIPI community onZenodo? Briefly discussed this
with Henk as there will be a software release for teh challenge as well and
presumaably others. Would be great if these all come under the same
community

Prof. Steven Sourbron, PhD
Chair in Medical Imaging Physics
University of Sheffield, UK
*https
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>*
*Sheffield abdominal imaging research page*
<https://www.sheffield.ac.uk/medicine/research/research-themes/medical-imaging/medical-imaging-research/abdominal-imaging>
*Open source initiative in perfusion imaging* <http://www.osipi.org>
*Magnetic resonance imaging biomarkers for chronic kidney disease
<https://renalmri.org/>*

**Reply by petravanhoudt** [reply]
@HenkMutsaerts, do you know if we are the first that want to create something on Zenodo?

**Comment by petravanhoudt** [comment]
Hi @mjt320, I guess you need admin rights to the repository? Henk recently reorganized the rights and put everyone in teams, where members have write access. Only lead/colead + management team have admin rights at the moment. I added you as an individual member now with maintain rights. Are these sufficient or do you need a level higher to create the DOI? 

Let me know when you are done, such that we can remove the rights to keep things clean according to Henk's standards :)

Petra

**Comment by mjt320** [comment]
Thanks @petravanhoudt . That should work.
@LucyKershaw It stores a frozen snapshot of the repo and makes a DOI for it
@plaresmedima I can't see an OSIPI community on zenodo.

**Reply by mjt320** [reply]
I can create a zenodo group now and transfer the ownership later, if nobody objects.
Ultimately, it might be easiest to set up an osipi github account for this purpose (if there isn't already one)

**Reply by petravanhoudt** [reply]
sounds good!

**Comment by plaresmedima** [comment]
I think that's great but please check with Henk as he was looking at this
too. I have copied him in. Just so we don't end up with two ;-)

Prof. Steven Sourbron, PhD
Chair in Medical Imaging Physics
University of Sheffield, UK
*https
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>*
*Sheffield abdominal imaging research page*
<https://www.sheffield.ac.uk/medicine/research/research-themes/medical-imaging/medical-imaging-research/abdominal-imaging>
*Open source initiative in perfusion imaging* <http://www.osipi.org>
*Magnetic resonance imaging biomarkers for chronic kidney disease
<https://renalmri.org/>*

**Comment by notZaki** [comment]
For zenodo, will the co/authors be a catch-all like "OSIPI Community" or will it be every code contributor?

Also, if the citability of the zenodo DOI is important, then it might be worth adding a small "How to cite" section, such as a paragraph in the readme, to help out people that may be unfamiliar with zenodo. For example, there's usually multiple zenodo DOIs: each release gets its own and then there is one which always resolves to the latest release, so we could include the always-resolves-to-latest DOI in the readme to simplify it for users.

**Reply by mjt320** [reply]
Thanks @notZaki . I'm new to Zenodo so that's helpful.
The idea is to provide a DOI referencing the data presented in the paper, i.e. the first release. Authors would be all of us, I guess.
Agree it's important to consider how users can cite code and when to make future releases - probably something for the current TF & leadership to discuss @stadmill @LucyKershaw

**Comment by plaresmedima** [comment]
You can have an osipi community but then inside of that any number of code
releases or data each with their own set of authors.

For this I think you should also use a versioned DOI so that the same DOI
can be used to refer to any future versions as well.

As an example we just set a community up for Sheffield

https://zenodo.org/communities/qib-sheffield/?page=1&size=20

It contains one software release with a versioned DOI cited in the
accompanying paper

https://zenodo.org/record/7507642#.ZAo6caTLc0E
